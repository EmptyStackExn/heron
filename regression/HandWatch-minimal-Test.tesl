// HandWatch
// Minimal fast simulation

rational-clock sec
rational-clock min sporadic 0.0
unit-clock move
tag relation sec = 60.0 * min
min implies move
min time delayed by 1.0 on min implies min

// The policy `asap` drives the simulation where events are supposed
// to occur as fast as possible and in mininimal number of occurrence.
@policy asap

@step
@step
@step
@step

@output tikz select sec min move
