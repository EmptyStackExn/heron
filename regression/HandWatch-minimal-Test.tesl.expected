  \node[tick] at (1, -1)  {} ;
  \node[date] at (1, -1)  {0.} ;
  \node[date] at (1, 0)  {0.} ;
  \node[tick] at (1, -2)  {} ;
  \node[date] at (2, -1)  {1.} ;
  \node[tick] at (2, -1)  {} ;
  \node[date] at (2, 0)  {60.} ;
  \node[tick] at (2, -2)  {} ;
  \node[date] at (3, -1)  {2.} ;
  \node[tick] at (3, -1)  {} ;
  \node[date] at (3, 0)  {120.} ;
  \node[tick] at (3, -2)  {} ;
  \node[date] at (4, -1)  {3.} ;
  \node[tick] at (4, -1)  {} ;
  \node[date] at (4, 0)  {180.} ;
  \node[tick] at (4, -2)  {} ;
